interface-impl-amf/src/main/java/org/mule/amf/impl/model/QuotedJsonParameterValidationStrategy.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+package org.mule.amf.impl.model;
+
+import amf.client.model.domain.AnyShape;
+import amf.client.validate.ValidationReport;
+
+import static org.mule.amf.impl.model.MediaType.APPLICATION_JSON;
+import static org.mule.amf.impl.model.ParameterImpl.quote;
+
+class QuotedJsonParameterValidationStrategy extends JsonParameterValidationStrategy implements ParameterValidationStrategy {
+
+  QuotedJsonParameterValidationStrategy(AnyShape anyShape){
+    super(anyShape);
+  }
+
+  @Override
+  public ValidationReport validate(String value) {
+    return value == null ? nullValidationReport : super.validate(sanitize(value));
```

Comment:
I still think that having two different classes to differentiate quote and notQuote is useless.
Both classes that extend `JsonParameterValidationStrategy` have almost the same code:
`return value == null ? nullValidationReport : super.validate(THIS_IS_WHAT_VARIES)`
I would keep `JsonParameterValidationStrategy` only and add the quoting and sanitization code in there